src/kit/taosdump/taosdump.c

```diff
@@ -3054,3 +2713,154 @@ static int taosDumpIn() {
     return 0;
 }
 
+int main(int argc, char *argv[]) {
+    static char verType[32] = {0};
+    sprintf(verType, "version: %s\n", version);
+    argp_program_version = verType;
+
+    int ret = 0;
+    /* Parse our arguments; every option seen by parse_opt will be
+       reflected in arguments. */
+    if (argc > 1) {
+//        parse_precision_first(argc, argv, &g_args);
+        parse_timestamp(argc, argv, &g_args);
+        parse_args(argc, argv, &g_args);
+    }
+
+    argp_parse(&argp, argc, argv, 0, 0, &g_args);
+
+    if (g_args.abort) {
+#ifndef _ALPINE
+        error(10, 0, "ABORTED");
+#else
+        abort();
+#endif
+    }
+
+    printf("====== arguments config ======\n");
+    {
+        printf("host: %s\n", g_args.host);
+        printf("user: %s\n", g_args.user);
+        printf("password: %s\n", g_args.password);
+        printf("port: %u\n", g_args.port);
+        printf("mysqlFlag: %d\n", g_args.mysqlFlag);
+        printf("outpath: %s\n", g_args.outpath);
+        printf("inpath: %s\n", g_args.inpath);
+        printf("resultFile: %s\n", g_args.resultFile);
+        printf("encode: %s\n", g_args.encode);
+        printf("all_databases: %s\n", g_args.all_databases?"true":"false");
+        printf("databases: %d\n", g_args.databases);
+        printf("databasesSeq: %s\n", g_args.databasesSeq);
+        printf("schemaonly: %s\n", g_args.schemaonly?"true":"false");
+        printf("with_property: %s\n", g_args.with_property?"true":"false");
+        printf("avro format: %s\n", g_args.avro?"true":"false");
+        printf("start_time: %" PRId64 "\n", g_args.start_time);
+        printf("human readable start time: %s \n", g_args.humanStartTime);
+        printf("end_time: %" PRId64 "\n", g_args.end_time);
+        printf("human readable end time: %s \n", g_args.humanEndTime);
+        printf("precision: %s\n", g_args.precision);
+        printf("data_batch: %d\n", g_args.data_batch);
+        printf("max_sql_len: %d\n", g_args.max_sql_len);
+        printf("table_batch: %d\n", g_args.table_batch);
+        printf("thread_num: %d\n", g_args.thread_num);
+        printf("allow_sys: %d\n", g_args.allow_sys);
+        printf("abort: %d\n", g_args.abort);
+        printf("isDumpIn: %d\n", g_args.isDumpIn);
+        printf("arg_list_len: %d\n", g_args.arg_list_len);
+        printf("debug_print: %d\n", g_args.debug_print);
+
+        for (int32_t i = 0; i < g_args.arg_list_len; i++) {
+            printf("arg_list[%d]: %s\n", i, g_args.arg_list[i]);
+        }
+    }
```

Comment:
sorry, I might miss something but why do we need to use curly braces here?

Reply:
It's something personal favorite style to avoid add a new line in the loop but forget to add brace leads logical bug.